packages/components/src/switch/tests/chromatic/Switch.stories.tsx

```diff
@@ -213,3 +213,16 @@ export const CheckedStates: Story = {
         defaultSelected: true
     }
 };
+
+export const Overflow: Story = {
```

Comment:
for Switch, overflow is also available here and you can just reduce the width or something to improve on it https://6482172cea3f45dd1bb6ef3f-tzpkqamxoe.chromatic.com/?path=/story/components-forms-switch--unchecked